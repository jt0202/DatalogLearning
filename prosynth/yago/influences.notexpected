ion_negoi≈£escu	robert_brandom
stephen_king	allison_crowe
j_r_r_tolkien	gregory_maguire
richard_pryor	richard_yates_novelist
immanuel_kant	arliss_ryan
