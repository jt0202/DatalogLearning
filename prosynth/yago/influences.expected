liang_sicheng	lin_huiyin
